### PR TITLE
Step 3 of multipool RBE configuration.

### PR DESCRIPTION
**Pull request by adelez**

### CODE REVIEW

**Comment by grpc-testing** [comment]
```
****************************************************************

libgrpc.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]


****************************************************************

libgrpc++.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]



```

**Comment by grpc-testing** [comment]
```
[trickle] No significant performance differences
```

**Comment by grpc-testing** [comment]
```
[microbenchmarks] No significant performance differences
```

**Comment by grpc-testing** [comment]
```
****************************************************************

libgrpc.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]


****************************************************************

libgrpc++.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]



```

**Comment by grpc-testing** [comment]
```
[trickle] No significant performance differences
```

**Comment by grpc-testing** [comment]
```
[microbenchmarks] No significant performance differences
```

**Comment by adelez** [comment]
Bazel Debug build for C/C++: #14730
Bazel Opt build for C/C++: #14240

**Comment by grpc-testing** [comment]
```
****************************************************************

libgrpc.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]


****************************************************************

libgrpc++.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]



```

**Comment by grpc-testing** [comment]
```
[trickle] No significant performance differences
```

**Comment by grpc-testing** [comment]
```
[microbenchmarks] No significant performance differences
```

File: third_party/toolchains/BUILD [hunk 1]
```diff
@@ -39,8 +41,12 @@ platform(
           value:"docker://gcr.io/cloud-marketplace/google/rbe-ubuntu16-04@sha256:59bf0e191a6b5cc1ab62c2224c810681d1326bad5a27b1d36c9f40113e79da7f"
         }
         properties: {
-          name: "gceMachineType"
-          value: "n1-highmem-2"
+          name: "gceMachineType"  # Small machines for majority of tests.
+          value: "n1-standard-1"
```
Comment (matt-kwong) [inline third_party/toolchains/BUILD]: Does this mean that all RBE tests have been running on `n1-highmem-2`? My worry is that by downgrading the non-flaky tests to use `n1-standard-1`, we introduce flakes.

Comment (adelez) [reply third_party/toolchains/BUILD]: I did find two new tsan tests that fail to allocate memory in n1-standard-1, but work fine on n1-standard-8.

Comment (adelez) [reply third_party/toolchains/BUILD]: Based on the email thread with RBE team, I'll move forward with this change. We can roll back if unexpected failures/flakes occur.

**Pull request merged**